src/Microsoft.DotNet.Build.Tasks/PackageFiles/partialfacades.task.targets

```diff
@@ -41,12 +41,11 @@
     <!-- Move the PDB into a subdirectory for GenFacades if we are producing PDBs -->
     <Move SourceFiles="$(PartialFacadeSymbols)"
           DestinationFolder="$(GenFacadesInputPath)"
-          Condition="'$(DebugSymbols)' != 'false'"
-    />
+          Condition="'$(DebugSymbols)' != 'false' AND '$(OSGroup)' == 'Windows_NT'" />
```

Comment:
I would suggest moving the ProducePdb property above this and using that in the condition.